servant-client-ghcjs/README.md

```diff
@@ -0,0 +1,176 @@
+# servant-client-ghcjs
+
+Type safe querying of servant APIs from the browser.
+
+`servant-client-ghcjs` is much like `servant-client`, as both packages allow you to generate functions that query the endpoints of your servant API. Both packages should feel the same in usage. The big difference lies in how they perform the actual requests. `servant-client` (indirectly) uses your operating system's socket mechanisms, whereas `servant-client-ghcjs` uses your browser's [XHR](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest) mechanisms to send requests.
+
+This guide assumes knowledge of servant. Reading its [documentation](haskell-servant.readthedocs.io) is recommended if you're new to the subject.
```

Comment:
Please add http[s]:// before the link here as e.g if viewed on github, it just sends the reader to some file named "haskell-servant.readthedocs.io" in the LumiGuide repo =)